GVFS/GVFS.Installer.Mac/scripts/Distribution.xml

```diff
@@ -34,7 +34,7 @@
         
         function GetBlockingProcesses()
```

Comment:
Does this get called before or after `preinstall`?

Reply:
Before `preinstall`. 

This is the order - `Distribution.xml::InstallationCheck`, `preinstall`, `postinstall`.